### PR TITLE
extend ro values for hydrant_output and models + comment the hydrant_models tables

### PR DESCRIPTION
**Pull request by tudorbarascu**

### CODE REVIEW

**Comment by tudorbarascu** [comment]
Tested locally. Travis is a bad boy.

**Comment by haubourg** [comment]
And yes Travis is nasty, we have a new issue with it concerning Postgis Install and data sample auto dump and push to qwat-data-sample repo.  @3nids @sylvainbeo Anything new on fixing this?

**Comment by tudorbarascu** [comment]
@haubourg I added the delta file. Pushed the model to 1.2.7.
I guess this is how it's supposed to work, right?

**Comment by sylvainbeo** [comment]
That's new. Travis normally includes Postgis with Postgres installations...i have a look.

**Comment by m-kuhn** [comment]
@sylvainbeo already trying something over there: https://github.com/qwat/qwat-data-model/pull/181#issuecomment-312884547

**Comment by tudorbarascu** [comment]
@haubourg @3nids This is tested locally and it's safe. Can we merge it?

File: update/delta/delta_1.2.8_add_ro_values_and_hydrant_model_comments.sql [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+UPDATE qwat_vl.hydrant_model_inf SET value_ro = 'VonRoll_1989_5000' WHERE id = 7317;
+UPDATE qwat_vl.hydrant_model_inf SET value_ro = 'VonRoll_2007_5000' WHERE id = 7318;
+
+/* Add comments on the hydrant model inf/sup tables */
+
+COMMENT ON TABLE qwat_vl.hydrant_model_sup IS 'Holds the values for the ON THE ground hydrant models';
+
+COMMENT ON TABLE qwat_vl.hydrant_model_sup IS 'Holds the values for the underground hydrant models';
+
+UPDATE qwat_sys.versions SET version = '1.2.8';
```
Comment (haubourg) [inline update/delta/delta_1.2.8_add_ro_values_and_hydrant_model_comments.sql]: @3nids @sylvainbeo I think that we wanted now to keep track of all applied updates. This should that be an INSERT right?

**Comment by haubourg** [comment]
@tudorbarascu  Matthias and Mario are going to try to fix the travis part. 
I would like to wait tests to be run before merge, and have the automatically updated data sample at the same time. Can this wait next week?

File: update/delta/delta_1.2.8_add_ro_values_and_hydrant_model_comments.sql (hunk 1 above)
Comment (tudorbarascu) [inline update/delta/delta_1.2.8_add_ro_values_and_hydrant_model_comments.sql]: @haubourg I think delta should contain UPDATE and ``system/versions_insert.sql`` should contain INSERT of the model no.

**Comment by tudorbarascu** [comment]
@haubourg It can wait of course

**Comment by haubourg** [comment]
It's time to have @m-kuhn in the loop for those questions. 
until now, the version table only stored the current version. 
Good practices analyses concludes that it's good to keep track of all version upgrades that a DB has gone through, but this can wait the conclusions of Matthias and Mario before starting that in QWAT

**Comment by tudorbarascu** [comment]
@haubourg **until now, the version table only stored the current version.**
Yes, that's how it currently works, the INSERT with the data model is called on new data model creation and the UPDATE is called between upgrades (is inside the delta). I noticed this from Denis at #166

**Comment by lbartoletti** [comment]
@tudorbarascu can you update your PR for next release, please?

**Comment by tudorbarascu** [comment]
@lbartoletti This can also be merged.

**Comment by lbartoletti** [comment]
Thanks @tudorbarascu !

**Comment by lbartoletti** [comment]
target for release 1.3.3 #250

**Pull request merged**